lib/chef/data_bag.rb

```diff
@@ -70,11 +70,11 @@ def to_json(*a)
     end
 
     def chef_server_rest
-      @chef_server_rest ||= Chef::REST.new(Chef::Config[:chef_server_url])
+      @chef_server_rest ||= Chef::ServerAPI.new(Chef::Config[:chef_server_url])
```

Comment:
Yeah, I suspect this is another case where we're relying on some hidden side effect of how we require the universe in order to make everything work...